jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannel.java

```diff
@@ -511,12 +528,7 @@ public boolean handle()
                                 break;
                             }
                         }
-
-                        // TODO Currently a blocking/aborting consumeAll is done in the handling of the TERMINATED
-                        // TODO Action triggered by the completed callback below.  It would be possible to modify the
-                        // TODO callback to do a non-blocking consumeAll at this point and only call completed when
-                        // TODO that is done.
-
+                        
```

Comment:
The block above has a call to `abort()`, which then falls through, which I don't think it's right, we should `break`.

Reply:
Why is that not correct?   It still sets up the final call to completed.
Besides, that is existing behaviour and not really related to this PR.

Reply:
Okay.